__init__.py

```diff
@@ -59,6 +59,8 @@ class ExportGLTF(bpy.types.Operator, ExportHelper):
         check_extension = True
 
         #blendergltf settings
+        nodes_export_hidden = BoolProperty(name='Export Hidden Objects', default=False)
+        nodes_selected_only = BoolProperty(name='Selection Only', default=False)
```

Comment:
These settings also need to be added to the default_settings dictionary near the top of blendergltf.py.